app/code/Magento/Review/Service/GetReviewAverageRatingService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Review\Service;
+
+/**
+ * Get review average rating
+ */
+class GetReviewAverageRatingService
+{
+    /**
+     * Get average rating per review
+     *
+     * @param array $ratingVotes
+     *
+     * @return float
+     */
+    public function execute(array $ratingVotes): float
```

Comment:
In this logic extracted from some other method? If yes, please make sure we do not have code duplicating and reuse this class in place from where it is extracted.

Reply:
Actually there is no place where I've extracted this one. Moreover, there is a different approach to calculate the average, and probably will try to use the same approach.
However, I have a question here (just to make sure that the value is the expected one). Does this `average_rating` represent a percentage like `80%`, or is should be kind of `4` - being the average rating that for that review?
E.g. 
- Price 4 stars
- Quality 4 stars

The output should be?
1. 80
2. 4

Thank you.

Reply:
After the latest updated, I'm just returning here `4`. Please let me know if it needs to be changed.